packages/hooks/src/useWalletTransactions.ts

```diff
@@ -0,0 +1,314 @@
+import { useFetch } from '@deriv/api';
+import { Statement } from '@deriv/api-types';
```

Comment:
Shouldn't it be as a dependecy of the `hooks` package?

Reply:
@hamid-deriv if you mean useFetch, that no. it's part of `@deriv/api` package